src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/WebserviceController.php

```diff
@@ -409,4 +396,47 @@ private function getErrorMessages()
             DuplicateWebserviceKeyException::class => $this->trans('This key already exists.', 'Admin.Advparameters.Notification'),
         ];
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return array<string, bool|string|null>
+     */
+    private function getWebServiceStatus(Request $request): array
+    {
+        $webserviceConfiguration = $this->get('prestashop.admin.webservice.form_data_provider')->getData();
+        $webserviceStatus = [
+            'enabled' => $webserviceConfiguration['enable_webservice'],
+            'isFunctional' => true,
```

Comment:
IMHO isFunctional default value should be false

if the webservice is enabled, the functionning status will be chek at line 416. If it's not enbaled, it's not functional

Reply:
Good point.